devsetup/tripleo/ceph.sh

```diff
@@ -61,4 +61,6 @@ openstack overcloud ceph deploy \
     --ceph-spec ceph_spec.yaml \
     --network-data network_data.yaml \
     --cephadm-default-container \
-    --output deployed_ceph.yaml
+    --output deployed_ceph.yaml \
+    --config initial-ceph.conf \
+    --single-host-defaults # TODO: jgilaber only required have ceph working with only 1 compute node should remove it once we move to have 3 computes
```

Comment:
`--single-host-defaults`  represents a workaround for a 3+1 scenario.
I'm ok with the rest of the patch but I'm wondering if we want to merge the workaround, or hold this patch until a 3+3 topology (that doesn't need this) is in place.

Reply:
@fmount our plan (from CI side) was to merge this workaround clearly marked as such, so we can set a periodic job already to make sure changes anywhere do not break it (or we can tell early if they do). And then soon come back and remove the workaround to setup the 3+3 scenario

Reply:
Ack, so +1 from me: the only thing I'm not sure is if `--single-host-defaults` is required in this contex (not sure you have the chance to test it)

Reply:
sure, can verify that later today, once the current run finishes

Reply:
can confirm that `--single-host-defaults` is not needed, build https://review.rdoproject.org/zuul/build/65131a41dae04ce3bd8bfab367d594d2 passed without it, as can be seen in https://logserver.rdoproject.org/09/53209/9/check/data-plane-adoption-osp17-multinode-to-extracted-crc-with-ceph/65131a4/controller/ci-framework-data/logs/ci_script_010_run_tripleo.log, the ceph deploy command used is `openstack overcloud ceph deploy --tld localdomain --ceph-spec ceph_spec.yaml --network-data network_data.yaml --cephadm-default-container --output deployed_ceph.yaml --config initial-ceph.conf`

Reply:
+1